pkg/config/engine/crio/option.go

```diff
@@ -17,17 +17,14 @@
 package crio
 
 import (
-	"fmt"
-	"os"
-
-	"github.com/pelletier/go-toml"
-
 	"github.com/NVIDIA/nvidia-container-toolkit/internal/logger"
+	"github.com/NVIDIA/nvidia-container-toolkit/pkg/config/toml"
 )
 
 type builder struct {
-	logger logger.Interface
-	path   string
+	logger    logger.Interface
+	reference toml.Loader
```

Comment:
I feel like the name `reference` may be a bit too vague. How about `configRef`?

Reply:
Sure.

Reply:
What about `tomlSource`?